ReactiveArchitecture/Classes/core/AppDelegate/AppDelegate.swift

```diff
@@ -36,29 +36,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         return true
     }
-
-    func applicationWillResignActive(_ application: UIApplication) {
-        // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
-        // Use this method to pause ongoing tasks, disable timers, and invalidate graphics rendering callbacks. Games should use this method to pause the game.
-    }
```

Comment:
Line length and general boilercrap cleanup

Reply:
:thumbsup: - on the android side I knew the analysis too.ls to enforce this. Not the case on iOS swift...yet.